Source/Lib/Codec/EbEncHandle.c

```diff
@@ -2218,9 +2223,15 @@ void CopyApiFromApp(
         sequenceControlSetPtr->staticConfig.lookAheadDistance = ComputeDefaultLookAhead(&sequenceControlSetPtr->staticConfig);
     }
 
-
-
-
+    //Set required flags to signal vbv status when hrd is enabled
+    if (sequenceControlSetPtr->staticConfig.hrdFlag == 1) {
+		sequenceControlSetPtr->staticConfig.videoUsabilityInfo = 1;
```

Comment:
Please fix the indent of your code changes. Use space instead of tab.

Reply:
Modified code , Used space in place of tab, reviewed full EbEncHandle.c file, Commit : 5d0af0f